docs/src/main/paradox/user/lang/kotlin/gettingstarted.md

```diff
@@ -0,0 +1,288 @@
+# Getting started with stateful services in Kotlin
+
+## Prerequisites
+
+Kotlin version
+: Cloudstate Kotlin support requires at least Java 8, though we recommend using Java 11, which has better support for running in containers. While it is possible to build a GraalVM native image for Cloudstate Kotlin user functions, at this stage Cloudstate offers no specific assistance or configuration to aid in doing that.
+
+Build tool
+: Cloudstate does not require any particular build tool, you can select your own. Or use [Cloudstate CLI](https://github.com/sleipnir/cloudstate-cli)
+
+protoc
+: Since Cloudstate is based on gRPC, you need a protoc compiler to compile gRPC protobuf descriptors. While this can be done by downloading, installing and running protoc manually, most popular build tools have a protoc plugin which will automatically compile protobuf descriptors during your build for you.
+
+docker
+: Cloudstate runs in Kubernetes with [Docker], hence you will need Docker to build a container that you can deploy to Kubernetes. Most popular build tools have plugins that assist in building Docker images.
+
+In addition to the above, you will need to install the Cloudstate Kotlin support library, which can be done as follows:
+    
+Maven
+: @@@vars
+```xml
+<dependency>
+  <groupId>io.cloudstate</groupId>
+  <artifactId>cloudstate-kotlin-support</artifactId>
+  <version>$cloudstate.kotlin-support.version$</version>
+</dependency>
+```
+@@@
+  
+sbt
+: @@@vars
+```scala
+libraryDependencies += "io.cloudstate" % "cloudstate-kotlin-support" % "$cloudstate.kotlin-support.version$"
+```
+@@@
+
+gradle
+: @@@vars
+```groovy
+compile group: 'io.cloudstate', name: 'cloudstate-kotlin-support', version: '$cloudstate.kotlin-support.version$'
+```
+@@@
+
+## Maven example
+
+A minimal Maven example pom file, which uses the [Xolstice Maven Protocol Buffers Plugin](https://www.xolstice.org/protobuf-maven-plugin/) and the [Google Jib Docker Maven Plugin](https://github.com/GoogleContainerTools/jib/), for a shopping cart service, is shown below:
+
+Remember to change the values of the main.class, repo.name, and version tags to their respective values
```

Comment:
you could use a `@@@ note { title=Important }` here perhaps (as the pom file doe not follow right after the ending ":")